### PR TITLE
[do not merge] Add nullbitsco tidbit board with pin mapping; doc updates; hex composer

### PR DESCRIPTION
**Pull request by patricksurry**
The hex_compose module is experimental keyboard-side composer for ascii/unicode characters which i'll use to drive a retro pc.   I can separate into a different PR if that's helpful

### CODE REVIEW

File: boards/nullbitsco/tidbit/main.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from kmk.keys import KC
+
+# add active_encoders=[0, 2] to constructor if first and third encoders installed
+keyboard = KMKKeyboard()
+
+XXXXX = KC.NO
+
+keyboard.keymap = [
+    [
+        XXXXX,  KC.PSLS, KC_PAST, KC_PMNS,
```
Comment (jaygreco) [inline boards/nullbitsco/tidbit/main.py:11]: These will all need to be converted from QMK syntax (underscore) to KMK (dot):
```
keyboard.keymap = [
    [
        XXXXX,  KC.PSLS, KC.PAST, KC.PMNS,
        KC.P7,  KC.P8,   KC.P9,   KC.PPLS,
        KC.P4,  KC.P5,   KC.P6,   KC.PPLS,
        KC.P1,  KC.P2,   KC.P3,   KC.PENT,
        KC.P0,  KC.P0,   KC.PDOT, KC.PENT,
    ]
]
```

**Review by jaygreco — COMMENTED** [review]
Looks good! There is only one change (keycode syntax) I needed to make to run this on my development TIDBIT with Bit-C PRO.

**Comment by patricksurry** [comment]
Splitting this into smaller PRs.  https://github.com/KMKfw/kmk_firmware/pull/916 has the vanilla tidbit stuff with Jay's suggestions.

**Comment by xs5871** [comment]
There are significant overlaps between #916 and #918. What do you want me to do here? Would you like to split those into atomic, potentially dependent PRs? Also, please close PRs you don't want to be considered for merging.

**Comment by patricksurry** [comment]
Sorry, yes I will close this PR.  Please go ahead and merge 916 (if it looks ok) and I'll rebase 918 and resubmit

**Pull request closed without merging**